### ISSUE TITLE
circleci: failed tests (14390): TestRetryableError 

### ISSUE THREAD

**Issue by tschottdorf** [cockroachdb/cockroach#5185]
The following test appears to have failed:

[#14390](https://circleci.com/gh/cockroachdb/cockroach/14390):

```
=== RUN   TestSendToOneClient
--- PASS: TestSendToOneClient (0.27s)
=== RUN   TestRetryableError
E160312 01:29:37.628946 rpc/context.go:155  rpc error: code = 4 desc = "context deadline exceeded"
I160312 01:29:37.634243 /go/src/google.golang.org/grpc/clientconn.go:318  Failed to dial 127.0.0.1:59663: grpc: the client connection is closing; please retry.
--- FAIL: TestRetryableError (5.22s)
	send_test.go:143: &{127.0.0.1:59663 127.0.0.1 {{} <nil> <nil> 0xc820102200 false false { <nil> [0xc82018fb80] 3000000000}}} has unexpectedly shut down
	<autogenerated>:31: Leaked goroutine: goroutine 376 [chan receive]:
		github.com/cockroachdb/cockroach/rpc.NewContext.func1()
			/go/src/github.com/cockroachdb/cockroach/rpc/context.go:99 +0x68
		github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker.func1(0xc82040a2a0, 0xc820418d60)
			/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:98 +0x60
		created by github.com/cockroachdb/cockroach/util/stop.(*Stopper).RunWorker
			/go/src/github.com/cockroachdb/cockroach/util/stop/stopper.go:99 +0x70
	<autogenerated>:31: Leaked goroutine: goroutine 377 [chan receive]:
		github.com/cockroachdb/cockroach/util.ListenAndServeGRPC.func1()
--
I160312 01:30:37.060125 kv/dist_sender_server_test.go:594  Waiting for the txn commit
I160312 01:30:37.062924 storage/replica_command.go:458  auto-gc'ed 31937226 (1 intents)
I160312 01:30:37.071955 storage/replica_command.go:458  auto-gc'ed adbc2b3b (1 intents)
I160312 01:30:37.077010 storage/engine/rocksdb.go:149  closing in-memory rocksdb instance
--- PASS: TestPropagateTxnOnPushError (1.00s)
FAIL
FAIL	github.com/cockroachdb/cockroach/kv	61.460s
=== RUN   TestCombinable
--- PASS: TestCombinable (0.00s)
=== RUN   TestBatchSplit
--- PASS: TestBatchSplit (0.00s)
=== RUN   TestBatchRequestGetArg
--- PASS: TestBatchRequestGetArg (0.00s)
=== RUN   TestKeyNext
--- PASS: TestKeyNext (0.00s)
=== RUN   TestKeyPrefixEnd
--- PASS: TestKeyPrefixEnd (0.00s)

```
Please assign, take a look and update the issue accordingly.

**Comment by tamird** [comment]
can't repro this

**Comment by petermattis** [comment]
Probably because I fixed it in #5211. Didn't realize there was an open issue.
